webservices/common/models/base.py

```diff
@@ -16,18 +16,31 @@ def followers(self):
         return self.app.config['SQLALCHEMY_FOLLOWERS']
 
     @property
-    def replica_tasks(self):
-        return self.app.config['SQLALCHEMY_REPLICA_TASKS']
+    def follower_tasks(self):
+        return self.app.config['SQLALCHEMY_FOLLOWER_TASKS']
+
+    @property
+    def restrict_follower_traffic_to_tasks(self):
+        return self.app.config['SQLALCHEMY_RESTRICT_FOLLOWER_TRAFFIC_TO_TASKS']
+
+    @property
+    def use_follower(self):
+        # Check for read operations and configured followers.
+        use_follower = (not self._flushing and self.followers)
+
+        # Optionally restrict traffic to followers for only supported tasks.
+        if use_follower and self.restrict_follower_traffic_to_tasks:
+            use_follower = (
+                celery.current_task and
+                celery.current_task.name in self.replica_tasks
```

Comment:
D'oh, thanks for catching.  It does work, provided the names match obviously. :-)